Revit_Core_Engine/Query/ViewSolid.cs

```diff
@@ -35,15 +35,17 @@ public static partial class Query
         /****              Public methods               ****/
         /***************************************************/
 
+        [PreviousVersion("7.3", "BH.Revit.Engine.Core.Query.ViewSolid(Autodesk.Revit.DB.View)")]
         [Description("Returns a solid that represents the 3-dimensional extents of a given view.")]
         [Input("view", "View to compute the solids.")]
+        [Input("createIfUncropped", "Create solid if crop view property is inactive. New solid will be extended by maximum extends value in directions perpendicular to the view type.")]
```

Comment:
```suggestion
        [Input("createIfUncropped", "If false, the method will return null in case of missing crop box (i.e. the view is unlimited, so it cannot be represented by a solid). If true, uncropped views will produce an 'unlimited' solid roughly 1e+6 by 1e+6 in dimensions perpendicular to the view direction, with depth equal to view depth or 1e+4 in case of views without depth.")]
```